### ISSUE TITLE
[SMB] Unknown NT STATUS Errors from [MS-ERREF] - 0xc9000000 / 3372220416

### ISSUE THREAD

**Issue by brunorafaeli** [fortra/impacket#1603]
### Configuration  
impacket version:  0_9_23
Python version:  3.10

### Debug Output With Command String  
`powershell.exe -executionpolicy bypass -File "C:\/Windows\/MyFolder\/MyFile.ps1"`
```
File /usr/local/lib/python3.10/site-packages/impacket/smb.py, line 521, in __str__/n<class 'KeyError'>:3372220416'
```
The error happens because the code is trying to access an error_code that does not exist in the mapping of NTSTATUS that comes from [[MS-ERREF]: NTSTATUS Values](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-erref/596a1078-e883-4972-9bbc-49e60bebca55) 

The code 3372220416 is 0xc9000000 in Hex, and is probably being converted by Python.

```
def __str__( self ):
        error_class = SessionError.error_classes.get( self.error_class, None )
        if not error_class:
            error_code_str = self.error_code
            error_class_str = self.error_class
        else:
            error_class_str = error_class[0]
            error_code = error_class[1].get( self.error_code, None )
            if not error_code:
                error_code_str = self.error_code
            else:
                error_code_str = '%s(%s)' % error_code

        if self.nt_status:
            return 'SMB SessionError: %s(%s)' % nt_errors.ERROR_MESSAGES[self.error_code] ### THIS LINE IS RAISING AN EXCEPTION
        else:
            # Fall back to the old format
            return 'SMB SessionError: class: %s, code: %s' % (error_class_str, error_code_str)
```

**Comment by anadrianmanrique** [comment]
Can you provide steps necessary to reproduce the issue? thanks

**Comment by rtpt-erikgeiser** [comment]
Sounds like it would be fixed by #1311.
